cmds/asset.js

```diff
@@ -117,5 +129,16 @@ exports.handler = (argv, _testTjmFunctions) => {
             return Promise.each(tjmConfig.clusters, cluster => latestAssetVersion(cluster));
         }
         return latestAssetVersion(tjmConfig.cluster);
+    } else if (argv.cmd === 'replace') {
```

Comment:
We should probably output a warning, saying that is potentially dangerous and is recommended to be used in development.